substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/ClassEntry.java

```diff
@@ -294,7 +294,9 @@ protected void processMethod(DebugMethodInfo debugMethodInfo, DebugInfoBase debu
          * substitution
          */
         FileEntry methodFileEntry = debugInfoBase.ensureFileEntry(fileName, filePath, cachePath);
-        methods.add(new MethodEntry(methodFileEntry, methodName, this, resultType, paramTypeArray, paramNameArray, modifiers));
+        final MethodEntry methodEntry = new MethodEntry(methodFileEntry, methodName, this, resultType, paramTypeArray, paramNameArray, modifiers);
```

Comment:
We generally do use `final` only for fields (on locals it's mostly verbose). But you can keep it if you like.